content/en/users/compute/orchestration/im/dashboard/_index.md

```diff
@@ -135,7 +135,9 @@ infrastructures:
 - Add nodes: The Add nodes action enables to add new VMs to the users'
   deployment. It will show the list of the different types of nodes currently
   deployed in the infrastructure and the user must set the number of nodes of
-  each type he wants to deploy.
+  each type he wants to deploy. It will also show the image selected to create
+  the VMs (if available). In the dropdown list the user can change the base
+  image, to use a new one or because the old one has been removed.
```

Comment:
This sentence reads a bit weird. Could you clarify?

Reply:
Is it clear now?